### ISSUE TITLE
markdown preview renders with black text

### ISSUE THREAD

**Issue by amithm7** [Microsoft/vscode#72927]
- VSCode Version: 1.33.1
- OS Version: Linux x64 4.9.0-8-amd64 (Debian Stretch)

Steps to Reproduce:

1. Open a markdown file.
2. Open preview to the side.

![md-dark-text](https://user-images.githubusercontent.com/7036770/56820399-6f51d700-6869-11e9-9895-c53ab190daa6.png)

As shown in the image here, the preview is rendered with black text, making it difficult to read with a dark background. Also the the scrollbar in the preview is not visible.
Only after 5-6 attempts (usually) does it render properly, other times it is like as shown in the image.

Does this issue occur when all extensions are disabled?: Yes

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/)?

**Comment by amithm7** [comment]
No, it doens't seem to be happening in insiders build.

**Comment by mjbvz** [comment]
Thanks. I believe this is a duplicate of #71741 which should be fixed in the stable April release
